components/Page/Header.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div>
+    <header class="page-header">
+      <!-- TODO: Only show for screen-readers -->
+      <div class="page-header__brand">
+        <h1 class="page-header__brand__text">
+          <nuxt-link to="/" class="label-ae">
+            <AtomLogo class="label-ae__logo"/>
+            <span class="label-ae__name">Appje-Eitje</span>
+            <span class="label-ae__service"></span>
+          </nuxt-link>
+        </h1>
+        <!-- <p class="page-header__brand__slogan">
```

Comment:
Waarom uitgecommented en niet verwijderd?

Reply:
Eigenlijk is onze bedrijfsnaam al een naam en slogan in een :-)